.nvmrc

```diff
@@ -1 +1 @@
-v6.10.2
+6.10.2
```

Comment:
Why did this change? Currently this should fail `yarn test` as the version verification script would compare against `process.version` which returns the version prefixed with `v`.